diff -aurp R-devel-orig/src/gnuwin32/Makefile R-devel/src/gnuwin32/Makefile
--- R-devel-orig/src/gnuwin32/Makefile	2019-11-20 12:43:22.000000000 +0100
+++ R-devel/src/gnuwin32/Makefile	2019-11-21 17:19:35.000000000 +0100
@@ -145,6 +145,8 @@ fixfiles: Rpwd.exe
 ifdef USE_LIBCURL
 	@if test -f "$(CURL_PATH)/etc/curl-ca-bundle.crt"; then \
 	  $(CP) -p "$(CURL_PATH)/etc/curl-ca-bundle.crt" ../../etc; \
+	elif test -f "$(CURL_CA_BUNDLE)"; then \
+	  $(CP) -p "$(CURL_CA_BUNDLE)" ../../etc/curl-ca-bundle.crt; \
 	fi
 endif
 
diff -aurp R-devel-orig/src/gnuwin32/MkRules.rules R-devel/src/gnuwin32/MkRules.rules
--- R-devel-orig/src/gnuwin32/MkRules.rules	2019-11-20 12:43:22.000000000 +0100
+++ R-devel/src/gnuwin32/MkRules.rules	2019-11-21 17:27:12.000000000 +0100
@@ -23,6 +23,7 @@ CAIRO_HOME ?=
 USE_CAIRO ?= $(if $(CAIRO_HOME),YES)
 CAIRO_CPPFLAGS ?= "-I$(CAIRO_HOME)/win$(WIN)"
 CAIRO_LIBS ?= "-L$(CAIRO_HOME)/win$(WIN) -lcairo -lpixman-1 -lpng -lz -lgdi32 -lmsimg32"
+CURL_CA_BUNDLE ?= /usr/ssl/certs/curl-ca-bundle.crt
 BUILD_HTML ?= NO
 MIKTEX ?= TRUE
 TEXI2ANY ?= missing
