diff -aurp R-devel-orig/src/gnuwin32/Makefile R-devel/src/gnuwin32/Makefile
--- R-devel-orig/src/gnuwin32/Makefile	2018-06-28 14:03:06.000000000 +0200
+++ R-devel/src/gnuwin32/Makefile	2018-06-29 13:32:13.000000000 +0200
@@ -42,9 +42,9 @@ rpackages:
 
 
 cairodevices:
-ifneq "$(CAIRO_HOME)" ""
+ifneq "$(USE_CAIRO)" ""
 	@$(MK) -C ../library/grDevices/src/cairo -f Makefile.win all \
-	  WIN=$(WIN) CAIRO_HOME=$(CAIRO_HOME)
+	  WIN=$(WIN) CAIRO_LIBS=$(CAIRO_LIBS) CAIRO_CPPFLAGS=$(CAIRO_CPPFLAGS)
 endif
 
 ifeq ($(wildcard MkRules?local),MkRules.local)
diff -aurp R-devel-orig/src/gnuwin32/MkRules.rules R-devel/src/gnuwin32/MkRules.rules
--- R-devel-orig/src/gnuwin32/MkRules.rules	2018-06-28 14:03:06.000000000 +0200
+++ R-devel/src/gnuwin32/MkRules.rules	2018-06-29 13:31:03.000000000 +0200
@@ -20,6 +20,9 @@ WIN ?= 32
 ICU_PATH ?= 
 ICU_LIBS ?= -lsicuin -lsicuuc -lsicudt -lstdc++
 CAIRO_HOME ?=
+USE_CAIRO ?= $(if $(CAIRO_HOME),YES)
+CAIRO_CPPFLAGS ?= "-I$(CAIRO_HOME)/win$(WIN)"
+CAIRO_LIBS ?= "-L$(CAIRO_HOME)/win$(WIN) -lcairo -lpixman-1 -lpng -lz -lgdi32 -lmsimg32"
 BUILD_HTML ?= NO
 MIKTEX ?= TRUE
 TEXI2ANY ?= missing
diff -aurp R-devel-orig/src/library/grDevices/src/cairo/Makefile.win R-devel/src/library/grDevices/src/cairo/Makefile.win
--- R-devel-orig/src/library/grDevices/src/cairo/Makefile.win	2018-06-28 14:03:06.000000000 +0200
+++ R-devel/src/library/grDevices/src/cairo/Makefile.win	2018-06-29 13:30:39.000000000 +0200
@@ -17,10 +17,9 @@ R_ARCH = /i386
 endif
 
 PKG_CPPFLAGS = -I$(R_HOME)/src/include -I. -I.. -DHAVE_CONFIG_H \
-  $(arch_DEFS) -I"$(CAIRO_HOME)/win$(WIN)"
+  $(arch_DEFS) $(CAIRO_CPPFLAGS)
 
-PKG_LIBS = -L"$(CAIRO_HOME)/win$(WIN)" -lcairo -lpixman-1 -lpng -lz \
-  -lgdi32 -lmsimg32 -L.. -lgrDevices
+PKG_LIBS = $(CAIRO_LIBS) -L.. -lgrDevices
 
 SOURCES = cairoBM.c
 DEPENDS = $(SOURCES:.c=.d)
