diff -aurp R-3.5.1-orig/src/gnuwin32/Makefile R-3.5.1/src/gnuwin32/Makefile
--- R-3.5.1-orig/src/gnuwin32/Makefile	2018-07-14 23:20:42.000000000 +0200
+++ R-3.5.1/src/gnuwin32/Makefile	2018-07-24 23:35:39.000000000 +0200
@@ -228,7 +228,7 @@ htmldocs:
 #                          ===== BLAS ======
 
 Rblas:
-	@$(MK) -C ../extra/blas -f Makefile.win FFLAGS='$(FFLAGS)'
+	@$(MK) -C ../extra/blas -f Makefile.win FFLAGS='-O2 $(EOPTS)'
 
 Rblas-clean:
 	@$(MK) -C ../extra/blas -f Makefile.win clean
diff -aurp R-3.5.1-orig/src/gnuwin32/fixed/Makefile R-3.5.1/src/gnuwin32/fixed/Makefile
--- R-3.5.1-orig/src/gnuwin32/fixed/Makefile	2018-07-14 23:20:42.000000000 +0200
+++ R-3.5.1/src/gnuwin32/fixed/Makefile	2018-07-24 16:27:47.000000000 +0200
@@ -44,6 +44,7 @@ ifeq "$(WIN)" "64"
 	  etc/Makeconf > $(R_HOME)/etc$(R_ARCH)/Makeconf
 else
 	$(SED) -e "s|IMPDIR = bin|IMPDIR = $(IMPDIR)|" \
+	  -e "s/-O3/-O2/" \
 	  -e "s/@EOPTS@/$(EOPTS)/" \
 	  -e "s|BINPREF =|BINPREF ?= $(BINPREF)|" \
 	  -e "s|COMPILED_BY =|COMPILED_BY = $(COMPILED_BY)|" \
