diff -aurp R-devel-orig/src/gnuwin32/fixed/h/config.h R-devel/src/gnuwin32/fixed/h/config.h
--- R-devel-orig/src/gnuwin32/fixed/h/config.h	2019-12-03 12:56:34.000000000 +0100
+++ R-devel/src/gnuwin32/fixed/h/config.h	2019-12-03 12:57:34.000000000 +0100
@@ -1133,7 +1133,7 @@
 
 /* Workaround for win64 pow() precision issue in Mingw-w64 V3 and higher
    See http://sourceforge.net/p/mingw-w64/bugs/466 for discussion. */
-#if defined(_WIN64) && defined(__MINGW64_VERSION_MAJOR) && __MINGW64_VERSION_MAJOR >= 3
+#if defined(__MINGW64_VERSION_MAJOR) && __MINGW64_VERSION_MAJOR >= 3
 #define USE_POWL_IN_R_POW 1
 #endif
 
