diff -aurp R-devel/src/gnuwin32/fixed/etc/Makeconf R-devel-orig/src/gnuwin32/fixed/etc/Makeconf
--- R-devel/src/gnuwin32/fixed/etc/Makeconf	2019-08-16 01:15:04.000000000 +0200
+++ R-devel-orig/src/gnuwin32/fixed/etc/Makeconf	2019-10-12 14:51:35.000000000 +0200
@@ -81,14 +81,14 @@ CXX11 = $(BINPREF)g++ $(M_ARCH)
 CXX11FLAGS = -O2 -Wall $(DEBUGFLAG) @EOPTS@
 CXX11PICFLAGS =
 CXX11STD = -std=gnu++11
-CXX14 = 
-CXX14FLAGS = 
+CXX14 = $(BINPREF)g++ $(M_ARCH)
+CXX14FLAGS = -O2 -Wall $(DEBUGFLAG) @EOPTS@
 CXX14PICFLAGS =
-CXX14STD = 
-CXX17 = 
-CXX17FLAGS = 
+CXX14STD = -std=gnu++14
+CXX17 = $(BINPREF)g++ $(M_ARCH)
+CXX17FLAGS = -O2 -Wall $(DEBUGFLAG) @EOPTS@
 CXX17PICFLAGS =
-CXX17STD = 
+CXX17STD = -std=gnu++17
 DYLIB_EXT = .dll
 DYLIB_LD = $(DLL)
 DYLIB_LDFLAGS = -shared
diff -aurp R-devel/src/gnuwin32/installer/Makefile R-devel-orig/src/gnuwin32/installer/Makefile
--- R-devel/src/gnuwin32/installer/Makefile	2019-09-28 01:15:03.000000000 +0200
+++ R-devel-orig/src/gnuwin32/installer/Makefile	2019-10-12 14:50:57.000000000 +0200
@@ -103,7 +103,7 @@ imagedir:
 	(cd ..; $(MAKE) --no-print-directory PKGDIR=. pkgclean-windlgs)
 	$(CP) -pR ../windlgs $(RPREFIX)/src/library
 # Edit Makeconf
-	$(SED) -e 's!^BINPREF [?]= .*!BINPREF ?= c:/Rtools/mingw_64/bin/!' \
+	$(SED) -e 's!^BINPREF [?]= .*!BINPREF ?= /mingw64/bin/!' \
 	  $(R_HOME)/etc/x64/Makeconf > $(RPREFIX)/etc/x64/Makeconf
 # test dir
 	mkdir $(RPREFIX)/tests
@@ -147,7 +147,7 @@ ifneq ($(strip $(HOME32)),)
              > $(RPREFIX)/library/$${f}/DESCRIPTION; \
 	done
 # Edit Makeconf file to default Rtools location
-	$(SED) -e 's!^BINPREF [?]= .*!BINPREF ?= c:/Rtools/mingw_32/bin/!' \
+	$(SED) -e 's!^BINPREF [?]= .*!BINPREF ?= /mingw32/bin/!' \
 	  $(HOME32)/etc/i386/Makeconf > $(RPREFIX)/etc/i386/Makeconf
 else
 	@echo "------------ not including 32-bit files"
