diff --git a/src/gnuwin32/installer/Makefile b/src/gnuwin32/installer/Makefile
index 297b1c22d5..3172090779 100644
--- a/src/gnuwin32/installer/Makefile
+++ b/src/gnuwin32/installer/Makefile
@@ -103,7 +103,7 @@ imagedir:
 	(cd ..; $(MAKE) --no-print-directory PKGDIR=. pkgclean-windlgs)
 	$(CP) -pR ../windlgs $(RPREFIX)/src/library
 # Edit Makeconf
-	$(SED) -e 's!^BINPREF [?]= .*!BINPREF ?= c:/Rtools/mingw_64/bin/!' \
+	$(SED) -e 's!^BINPREF [?]= .*!BINPREF ?= /mingw64/bin/!' \
 	  $(R_HOME)/etc/x64/Makeconf > $(RPREFIX)/etc/x64/Makeconf
 # test dir
 	mkdir $(RPREFIX)/tests
@@ -147,7 +147,7 @@ ifneq ($(strip $(HOME32)),)
              > $(RPREFIX)/library/$${f}/DESCRIPTION; \
 	done
 # Edit Makeconf file to default Rtools location
-	$(SED) -e 's!^BINPREF [?]= .*!BINPREF ?= c:/Rtools/mingw_32/bin/!' \
+	$(SED) -e 's!^BINPREF [?]= .*!BINPREF ?= /mingw32/bin/!' \
 	  $(HOME32)/etc/i386/Makeconf > $(RPREFIX)/etc/i386/Makeconf
 else
 	@echo "------------ not including 32-bit files"
