--- R-devel/src/library/tcltk/src/Makefile.win	2018-06-24 23:55:02.403619700 +0000
+++ R-devel/src/library/tcltk/src/Makefile.win.new	2018-06-24 23:55:27.395537100 +0000
@@ -21,12 +21,8 @@ RESFLAGS = -I $(R_HOME)/include
 
 ## The TCL_* variables are set in etc/Makeconf.
 ## Because the 64-bit build is customized, no other Tcl/Tk build is suitable
-PKG_CPPFLAGS = -I "$(TCL_HOME)"/include -DWin32 -I$(R_HOME)/src/include -DHAVE_CONFIG_H
-ifeq "$(WIN)" "64"
-PKG_LIBS = -L"$(TCL_HOME)"/bin64 -ltcl$(TCL_VERSION) -ltk$(TCL_VERSION)
-else
-PKG_LIBS = -L"$(TCL_HOME)"/bin -ltcl$(TCL_VERSION) -ltk$(TCL_VERSION)
-endif
+PKG_CPPFLAGS = -DSTATIC_BUILD -DWin32 -I$(R_HOME)/src/include -DHAVE_CONFIG_H
+PKG_LIBS = -ltk$(TCL_VERSION) -ltclstub$(TCL_VERSION) -ltcl$(TCL_VERSION) -lws2_32 -lnetapi32 -lgdi32 -luserenv -limm32 -lcomctl32 -loleaut32 -lole32 -lcomdlg32 -luuid
 
 all: makeMakedeps shlib
 
